libs/designer/src/lib/ui/templates/filters/templatesearchfilters.tsx

```diff
@@ -207,7 +207,7 @@ const Filters = ({ detailFilters }: { detailFilters: TemplateDetailFilterType })
       templateManifest.skus.includes(isConsumption ? 'consumption' : 'standard')
     );
     const allConnectors = Object.values(skuTemplates).flatMap((template) => template.featuredConnectors ?? []);
-    const uniqueConnectorsFromConnections = getUniqueConnectorsFromConnections(allConnectors, subscriptionId, location);
+    const uniqueConnectorsFromConnections = getUniqueConnectorsFromConnections(allConnectors, subscriptionId, location, true);
```

Comment:
Always put the argument name in comment for constant values like /* remove... */true